lib/home.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:meetup_chatapp/conversations_page.dart';
+import 'package:meetup_chatapp/options_page.dart';
+
+class HomePage extends StatefulWidget {
```

Comment:
What do you think about naming the file home_page.dart?  So far we seem to have fallen into that naming convention for other page widgets.

Reply:
Whoops, I think I got distracted. I'll fix it (and the splash page's name too) in another PR.